source/change-streams.rst

```diff
@@ -277,33 +306,48 @@ Driver API
      * This option is sent only if the caller explicitly provides a value. The
      * default is to not send a value.
      *
+     * @since 3.6
      * @see https://docs.mongodb.com/manual/reference/command/aggregate
      * @note this is an aggregation command option
      */
     collation: Optional<Document>;
 
+    /**
+     * The changeStream will only provides changes that occurred after the
+     * specified cluster timestamp. The default value is a cluster time obtained from
+     * the server before the changestream was created.
+     * @since 4.0
+     */
+    startAtClusterTime: Optional<Timestamp>
```

Comment:
Where would one get this Timestamp from?  If it's just a fancy Date, should we use Date instead?

Reply:
The timestamp is whatever value is returned as `$clusterTime` from an `ismaster`. Do we define that anywhere else?

Reply:
I see.  But where do we expect an application to get that value from?  Most drivers don't return the raw command result from CRUD methods.

Is the expectation that an application calls a command via RunCommand and grabs the cluster time from the response?  Or perhaps starts a session and retrieves it from the ClientSession?

Reply:
Internally, the driver should have a saved `$clusterTime` from an initial `ismaster`. As for how a user can manually set it, I do not know. I assume they need to run a command to obtain it.

I will update this to say that it is a `$clusterTime` and refer users to the `runCommand` documentation

Reply:
done